Tiltfile

```diff
@@ -208,3 +274,14 @@ if settings.get("deploy_cert_manager"):
 deploy_capi()
 
 capz()
+
+# run worker clusters specified from 'tilt up' or in 'tilt_config.json'
```

Comment:
Please wrap the lines below into a function. Perhaps, `flavors()`.

Also, please add a newline at the end of the file.

Reply:
Good catch!